news/2022-05-28-new-featured-artist-rameses-b.md

```diff
@@ -0,0 +1,147 @@
+- [Another osu! map](https://osu.ppy.sh/beatmapsets/300330) hosted by [Aka](https://osu.ppy.sh/users/1307553)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Timeless/Rameses%20B%20-%20Timeless%20(feat.%20Veela).mp3" type="audio/mpeg">
+</audio>
+
+### We Love
+
+Here's another two captivating maps:
+
+- [osu! map](https://osu.ppy.sh/beatmapsets/347276) hosted by [Come\[Back\]Home](https://osu.ppy.sh/users/1131716)
+- [Another osu! map](https://osu.ppy.sh/beatmapsets/239330) hosted by [Milan-](https://osu.ppy.sh/users/1052994)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/We%20Love/Rameses%20B%20-%20We%20Love.mp3" type="audio/mpeg">
+</audio>
+
+### Sweet Surrender (feat. Charlotte Haining)
+
+And here's [one more liquid drum & bass map](https://osu.ppy.sh/beatmapsets/358335) hosted by [Milan-](https://osu.ppy.sh/users/1052994)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Songs/Rameses%20B%20-%20Sweet%20Surrender%20(feat.%20Charlotte%20Haining).mp3" type="audio/mpeg">
+</audio>
+
+### We Are One (feat. Veela)
+
+Test your finger control with the doubles on [this osu! mapset](https://osu.ppy.sh/beatmapsets/522920) hosted by [Yamicchi](https://osu.ppy.sh/users/6486831)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Songs/Rameses%20B%20-%20We%20Are%20One%20(feat.%20Veela).mp3" type="audio/mpeg">
+</audio>
+
+### Samurai
+
+Try this [longer map](https://osu.ppy.sh/beatmapsets/1390484) hosted by [Dayno](https://osu.ppy.sh/users/10634465)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Samurai/Rameses%20B%20-%20Samurai.mp3" type="audio/mpeg">
+</audio>
+
+### Dream Catcher (feat. Charlotte Haining)
+
+Have fun with these:
+
+- [osu! map](https://osu.ppy.sh/beatmapsets/1174503) hosted by [Fall](https://osu.ppy.sh/users/4800816)
+- [osu!taiko map](https://osu.ppy.sh/beatmapsets/1033681) hosted by [gaston_2199](https://osu.ppy.sh/users/5938161)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Dream%20Catcher/Rameses%20B%20-%20Dream%20Catcher%20(feat.%20Charlotte%20Haining).mp3" type="audio/mpeg">
+</audio>
+
+### Transformations (feat. Laura Brehm)
+
+Congrats on making it to the end of the list. Here's [one final marathon](https://osu.ppy.sh/beatmapsets/593642) hosted by [Mir](https://osu.ppy.sh/users/8688812)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/270/Transformations/Rameses%20B%20-%20Transformations%20(feat.%20Laura%20Brehm).mp3" type="audio/mpeg">
+</audio>
+
+---
+
+That's a lot of songs, we know. And there's even more on [**Rameses B**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/270) waiting for you to begin your mapping journey. Don't keep them waiting.
+
+## Featured Artists in lazer
+
+Speaking of awesome new things, we're also experimenting with **Featured Artists in lazer playlists** starting TODAY.
```

Comment:
```suggestion
Speaking of awesome new things, we're also experimenting with **Featured Artists Playlists** in osu!(lazer) starting TODAY.
```

Probably reads better like this? I forget what we agreed on for mentions of lazer in news posts so might be worth checking previous usage.

Reply:
I'd rather have "Featured Artist playlists"

Reply:
sure